homeassistant/components/media_player/kodi.py

```diff
@@ -520,25 +564,31 @@ def supported_features(self):
         """Flag media player features that are supported."""
         supported_features = SUPPORT_KODI
 
-        if self._turn_off_action in TURN_OFF_ACTION:
+        if self._turn_on_action is not None:
+            supported_features |= SUPPORT_TURN_ON
+
+        if self._turn_off_action is not None:
             supported_features |= SUPPORT_TURN_OFF
 
         return supported_features
 
     @cmd
     @asyncio.coroutine
+    def async_turn_on(self):
+        """Execute turn_on_action to turn on media player."""
+        if self._turn_on_action is not None:
+            yield from self._turn_on_action.async_run()
```

Comment:
Ok, I got it, so, if changing the template in the warning to:
```yaml
turn_off_action:
  service: media_player.kodi_call_method
  data_template:
    entity_id: '{{ entity_id }}'
    method: System.Suspend
```
there is no need for using the `async_added_to_hass` coroutine, isn't it? I'll move the script creation to the setup again